Packs/Bitbucket/Integrations/Bitbucket/Bitbucket_description.md

```diff
@@ -0,0 +1,7 @@
+## Bitbucket Integration Help
+
+* A link to create an app password in Bitbucket: [link](https://developer.atlassian.com/cloud/bitbucket/rest/intro/#app-passwords)
```

Comment:
Explain here the steps how to create the app password and at the end add the link with further information (fix the markdown for the link as well)